app/controllers/api/v1/invites_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class Api::V1::InvitesController < Api::BaseController
+  include RegistrationHelper
+  include RoutingHelper
```

Comment:
I'm not sure this is necessary in a controller.

Reply:
Indeed. We do have a few controllers explicitly including it though.